app/views/candidate_interface/other_qualifications/review/show.html.erb

```diff
@@ -6,13 +6,13 @@
     <h1 class="govuk-heading-xl">
       <%= other_qualifications_title(@application_form) %>
     </h1>
-
-    <%= govuk_link_to t('application_form.other_qualification.another.button'), candidate_interface_other_qualification_type_path, button: true, class: 'govuk-button--secondary' %>
   </div>
 </div>
 
 <%= render(CandidateInterface::OtherQualificationsReviewComponent.new(application_form: @application_form)) %>
 
+<%= govuk_button_to t('application_form.other_qualification.another.button'), candidate_interface_other_qualification_type_path, class: 'govuk-button--secondary', method: :get %>
+
```

Comment:
Sorry, moving this button was in the prototype, but I’ve changed its location back to where it was before, above the list of qualifications